src/CookieRequest.jl

```diff
@@ -72,7 +72,7 @@ function getcookies(cookies, url)
 end
 
 function setcookies(cookies, host, headers)
-    for (k,v) in filter(x->x[1]=="Set-Cookie", headers)
+    for (k,v) in filter(x->lowercase(x[1])=="set-cookie", headers)
```

Comment:
```suggestion
    for (k,v) in filter(x->ascii_lc_isequal(x[1], "set-cookie"), headers)
```
we have an optimized function for comparing two ascii strings case insensitively